packages/core/src/context/ContextSystem.js

```diff
@@ -229,7 +229,10 @@ export class ContextSystem extends System
      */
     postrender()
     {
-        this.gl.flush();
+        if (this.renderer.renderingToScreen)
```

Comment:
just curious about why would _not_ want to call flush after rendering to a render texture?

Reply:
just extra gl operation in spector logs. Its you who started it!

Reply:
haha :D 
In you opinion, does it have any effect?

Reply:
Gl.flush() placed after main framebuffer render helped us on some mobile devices as browser/bug workaround. We didnt have it in v4. I have no idea how it affects things if we flush it after rendertextures

Reply:
cool!